lib/pymedphys/_experimental/streamlit/apps/transfer_check.py

```diff
@@ -97,23 +121,25 @@ def main():
         else:
             st.error("Name: " + name)
 
+        # Compare MRN between DICOM and MOSAIQ and write results
         if mrn == mosaiq_table.iloc[0]["mrn"]:
             st.success("MRN: " + mrn)
         else:
             st.error("MRN: " + mrn)
 
+        # Format DOB and compare between DICOM and MOSAIQ and write results
         DOB = str(mosaiq_table.iloc[0]["dob"])[0:10]
         dicom_DOB = dicom_table.iloc[0]["dob"]
         if DOB == dicom_DOB[0:4] + "-" + dicom_DOB[4:6] + "-" + dicom_DOB[6:8]:
             st.success("DOB: " + DOB)
         else:
             st.error("DOB: " + DOB)
 
-        ########################################################################################################################
-        # check the approval status of various sections in Mosaiq
+        ####################################################################################################################
+        # Section for checking approval statuses
```

Comment:
Opt for putting sections of code into either their own functions, or even their own files over comments like these.